internal/datasourcev2/streaming_data_source.go

```diff
@@ -304,8 +309,12 @@ func (sp *StreamProcessor) consumeStream(stream *es.Stream, closeWhenReady chan<
 	}
 }
 
-func (sp *StreamProcessor) subscribe(closeWhenReady chan<- struct{}) {
-	req, reqErr := http.NewRequest("GET", endpoints.AddPath(sp.cfg.URI, endpoints.StreamingRequestPath), nil)
+func (sp *StreamProcessor) subscribe(closeWhenReady chan<- struct{}, selector *fdv2proto.Selector) {
+	path := endpoints.AddPath(sp.cfg.URI, endpoints.StreamingRequestPath)
+	if selector.IsSet() {
+		path = path + "?basis=" + selector.State()
```

Comment:
I realize this is a hacky way of doing it. Just tried to get this working so I can continue with my test harness changes. But assuming you think this PR is correct overall, we can update this to use something more robust.